src/renderer/dashboard/index.jsx

```diff
@@ -106,6 +107,21 @@ const Dashboard = connect(mapStateToProps)(
 				<Grid container item direction="row" justify="flex-start" alignItems="flex-start">
 					<Typography variant="h1">SelfKey Dashboard</Typography>
 				</Grid>
+				<Grid
```

Comment:
Should this go away once application is approved?

Reply:
Good point.
I pushed a change to hide the widget if an existing application exists (even if it's not approved, because a user can only apply once).